app/views/contexts/summary/question.py

```diff
@@ -95,24 +108,91 @@ def _build_answers(
             ]
 
         summary_answers = []
-        for answer_schema in self.answer_schemas:
-            answer_value: Optional[AnswerValueEscapedTypes] = self.get_answer(
-                answer_store, answer_schema["id"]
-            )
-            answer = self._build_answer(
-                answer_store, question_schema, answer_schema, answer_value
-            )
 
-            summary_answer = Answer(
-                answer_schema=answer_schema,
-                answer_value=answer,
-                block_id=block_id,
-                list_name=list_name,
-                list_item_id=self.list_item_id,
-                return_to=return_to,
-                return_to_block_id=return_to_block_id,
-            ).serialize()
-            summary_answers.append(summary_answer)
+        if self.dynamic_answer_schemas:
```

Comment:
This should be simplified, instead of doing:
```py
if dynamic:
	process_normal()
	process_dynamic()
else:
	process_normal()
```

it should just be:

```py
if dynamic and list source:
	process_dynamic()

process_normal_if_any()
```